components/board/commonsysfs/ioctl.go

```diff
@@ -0,0 +1,168 @@
+package commonsysfs
+
+// This file is heavily inspired by https://github.com/mkch/gpio
```

Comment:
but why not just use why aren't we using https://github.com/mkch/gpio?

Reply:
That repo hasn't been updated since 2019, and its sole author has committed to Github on 14 days in the past year. I suspect it's not maintained. and given the frustrations I've had with Periph in the past month, if we need to change something later, I'd much rather it be in code we control.

Reply:
hmm it's a single dep so I'd just use it and if it comes time to change, just fork it under viamrobotics with a mod rename. no need to copy the code over now.